Duplicate rule: 'refer to' in RedHat.Usage and RedHat.see

Both RedHat.Usage and RedHat.see are displaying alerts  for 'refer to':

```
 158:122  suggestion  Verify your use of 'refer       RedHat.Usage          
                      to' with the word usage                               
                      guidelines.                                           
 158:122  error       Use 'see' rather than 'refer    RedHat.see            
                      to'.          
```

Keep only the RedHat.see alert.

--- a/.vale/styles/RedHat/Wordiness.yml
+++ b/.vale/styles/RedHat/Wordiness.yml
@@ -87,7 +87,7 @@ swap:
   it would appear that: apparently
   lift up: lift
   made reference to: referred to
-  make reference to: refer to
+  make reference to: see
   mix together: mix
   none at all: none
   not in a position to: unable

